app/models/machine.rb

```diff
@@ -223,6 +223,12 @@ class ResurrectionExpiredError < StandardError; end
       errors.add :fingerprint, :taken, message: "has already been taken for this product" if account.machines.joins(:product).exists?(fingerprint:, products: { id: product.id })
     when unique_per_policy?
       errors.add :fingerprint, :taken, message: "has already been taken for this policy" if account.machines.joins(:policy).exists?(fingerprint:, policies: { id: policy.id })
+    when unique_per_user?
+      errors.add :fingerprint, :taken, message: "has already been taken for this user" if
+        account.machines.exists?(
+          fingerprint:,
+          owner:,
```

Comment:
Feel like this needs to be scoped to the product so that machines across multiple products can work for the same owner.